tests/languages/kotlin/Tests.kt

```diff
@@ -133,6 +135,14 @@ class ServiceTest {
             )
             writeToFile(url)
 
+            // Multipart tests
+            val mp = general.multipart()
+
+            writeToFile((mp as Map<String, Any>)["x"] as String)
```

Comment:
What is the type of `mp`? Will developers also need to do all of those complex casts?

Reply:
It's not needed no, only